systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -757,7 +759,7 @@ void testForTopicOperator() throws InterruptedException {
         KafkaUtils.waitUntilKafkaCRIsReady(CLUSTER_NAME);
 
         assertThat(KafkaCmdClient.describeTopicUsingPodCli(CLUSTER_NAME, 0, "my-topic"),
-                hasItems("PartitionCount:2"));
+                hasItems("PartitionCount:", "2"));
```

Comment:
You should check only for one item `PartitionsCount:2`, I removed the whitespace in one of my recent PRs so it should work now.